scripts/release/asset_publish.py

```diff
@@ -317,15 +317,15 @@ def _str2bool(v: str) -> bool:
                 if env_label:
                     # TODO: Add fetching env from label
                     # https://github.com/Azure/azureml-assets/issues/415
-                    print("Unexpected !!! Registering a component with env label is still to be supported.")
+                    logger.print("Unexpected !!! Registering a component with env label is not yet supported.")
                     failure_list.append(asset)
                     continue
 
                 env = None
-                for version in set(env_version, final_version):
+                for version in set((env_version, final_version)):
```

Comment:
You probably want these to be ordered:
```suggestion
                for version in [env_version, final_version]:
```

Reply:
Done